modules/data-model/forms/impl/src/main/java/io/uhndata/cards/forms/internal/serialize/BareFormProcessor.java

```diff
@@ -58,6 +58,14 @@ public String getName()
         return "bare";
     }
 
+    @Override
+    public String getDescription()
+    {
+        return "Only runs on forms.\n"
```

Comment:
I would capitalize the initial of resource name and maybe also format it a bit differently to make it stand out.
```suggestion
        return "Only runs on `Forms`.\n"
```